src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/slurm/SlurmJobManager.groovy

```diff
@@ -138,136 +149,174 @@ class SlurmJobManager extends GridEngineBasedJobManager {
     /**
      * Reads the scontrol output and creates GenericJobInfo objects
      * @param resultLines - Input of ExecutionResult object
-     * @return map with jobid as key
+     * @return GenericJobInfo
      */
-    protected Map<BEJobID, GenericJobInfo> processExtendedOutput(String stdout, Map<BEJobID, GenericJobInfo> result) {
+    protected GenericJobInfo processExtendedOutput(String stdout) {
         // Create a complex line object which will be used for further parsing.
         ComplexLine line = new ComplexLine(stdout)
 
         Collection<String> splitted = line.splitBy(" ").findAll { it }
-        if (splitted.size() > 1) {
-            Map<String, String> jobResult = [:]
+        if (splitted.size() <= 1) {
```

Comment:
Maybe add a short comment, how the output looks like. I can only guess that this is some first element (what) that can be ignored, and some rest.